design/gateway-apis-implementation.md

```diff
@@ -62,69 +61,77 @@ Project Contour (the organisation) may investigate a Gateway API based layer 4 s
 
 ### Contour Gateway model
 
-Contour considers a Gateway to describe a single Envoy deployment, deployed and managed by something outside of itself.
-Contour expects to be supplied with the full name (name and namespace) of a single Gateway, which it will watch and update the status of.
-This will be a configuration file parameter.
-When a Gateway is not supplied, Contour will not do Gateway API processing, or in other words, the existing `--experimental-service-apis` flag will be obsoleted and removed).
-Inside Contour will merge Listeners within its Gateway. (See the detailed design for the exact rules Contour will use for this.)
+Contour considers a Gateway to describe a single Envoy deployment.
+Contour expects to be supplied with the full name and namespace of a single Gateway as well as the GatewayClass controller name.
+Contour will update the status of both the Gateway & GatewayClass.
+When a Gateway is not supplied, Contour will not do Gateway API processing.
```

Comment:
If we are moving to pass a `controllerName` instead of a Gateway name/namespace, we'll need to update this too:
```suggestion
When a GatewayClass `controllerName` is not configured, Contour will not do Gateway API processing.
```